sdk/cosmos/azure-cosmos-spark_3_2-12/src/main/scala/com/azure/cosmos/implementation/ChangeFeedSparkRowItem.scala

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.implementation
+
+import org.apache.spark.sql.Row
+
+private[cosmos] case class ChangeFeedSparkRowItem
```

Comment:
should this class just be an extend of the SparkRowItem?

Reply:
Tried that first - Case classes cannot use protected field - meaning you couldn't call the overloaded ctor of the base class to pass in the row. Wouldn't have much benefit anyway - so I kept them separate.